src/www/widgets/widgets/gateways.widget.php

```diff
@@ -78,6 +102,11 @@ function gateways_widget_update(sender, data)
         <tr>
             <th><?=gettext('Name')?></th>
             <th><?=gettext('RTT')?></th>
+<?php
+            if (isset($config['system']['prefer_dpinger'])) :?>                         
+            <th><?=gettext('STDEV')?></th>
```

Comment:
Should it be STDEV or RTTd / RTTD consistently for the visible GUI labels ?

Reply:
Well my original.. original... was RTTd, then someone said it should be STDEV... so which would you like, Personally I prefer RTTd

Reply:
Word! Let's make it RTTd then.

Reply:
So are you fixing this PR or do you want me to make any changes?

Little confused by the comment above

"it's ok as is now: return_apinger_defaults is a cheap call and is fixed up when prefer_dpinger is set below
the diff later also rewrites part of unrelated fixes on master"

Better I leave you to finish it if it's affecting other files.